path_to_ply : "/home/woo/catkin_ws/src/orchard_mapping/data/sample_plydata"
